src/zbBridgeAccessory.ts

```diff
@@ -69,10 +76,10 @@ export abstract class ZbBridgeAccessory {
       }
     });
 
-    this.platform.mqttClient.publish(
-      'cmnd/' + this.platform.mqttClient.topic + '/zbname',
-      this.addr + ',' + accessory.context.device.name,
-    );
+    // this.platform.mqttClient.publish(
```

Comment:
What’s the problem with this?

Reply:
It's because the Name is defined for whole device, not only for device+endpoint. But if endpoint is defined, then I need it represent with unique UUID, then I rename the device to name of first device+endpoint, after that to the second and so on. After all, it stay with last device+endpoint name is defined in config. I did not use device name, than it was easier for my to just block renaming than send few rename requests during each startup.